### ISSUE TITLE
“Wrong number of arguments: (4 . 4), 0” appears again

### ISSUE THREAD

**Issue by FaiChou** [clojure-emacs/cider#2743]
Tested on
```
elpa: 26.1
Cider version: 0.22.0 and 20191103.1528
flycheck-clojure version: 20190611.2351
flycheck version: 20191028.2247 
```

Opening `cider-jack-in` when in a `clj` file of a `clj` project, it always pop up errors when typing any character :

```
Wrong number of arguments: (4 . 4), 0
Error from syntax checker clojure-cider-eastwood: Done with no errors
```

**Comment by FaiChou** [comment]
Using [purcell](https://github.com/purcell/emacs.d)'s config, and old issues blow:

- https://github.com/clojure-emacs/squiggly-clojure/issues/29
- https://github.com/purcell/emacs.d/issues/502

**Comment by bbatsov** [comment]
I guess `squiggly-clojure` (`flycheck-clojure`) is out sync with CIDER. This issue should be fixed in the other project.

**Comment by zilti** [comment]
Every involved project in this chain just always refers to "the other guy has to fix it". This specific problem exists since years now.

**Comment by aarkerio** [comment]
I could fix it but through the [nrepl/lein-nrepl "0.3.2"] plugin.

https://gist.github.com/aarkerio/cbd3d020763d9e839afe7f13014fd3a4

Trying to connect Cider directly with the nrepl always gave me the  "clojure-cider-eastwood" error.
